### PR TITLE
Add --ignore-certificate-errors to skip not secure connection window

### PR DESCRIPTION
**Pull request by bartzet**
# Problem
image
blocking test executoion

# Solution
`--ignore-certificate-errors` needs to be added to `chromeOptions`

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [ ] Has the CHANGELOG been updated?
- [ ] Has the README been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-889-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-889-pr-onfido-sdk-ui-onfido.surge.sh

File: test/main.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ const random = () => Math.random().toString(36).substring(7)
 
 const chromeCapabilities = Capabilities.chrome()
 const chromeOptions = {
-  'args': ['--use-fake-device-for-media-stream','--use-fake-ui-for-media-stream', `--use-file-for-fake-video-capture=${__dirname}/resources/test-stream.y4m`]
+  'args': ['--use-fake-device-for-media-stream','--use-fake-ui-for-media-stream', '--use-file-for-fake-video-capture=${__dirname}/resources/test-stream.y4m', '--ignore-certificate-errors']
```
Comment (kopijunkie) [inline test/main.js]: `'--use-file-for-fake-video-capture=${__dirname}/resources/test-stream.y4m'` won't work after changing the back-tick ` to single quote '. the back-tick is needed to allow for the embedded parameter.

Comment (bartzet) [reply test/main.js]: Thanks!

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-889-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-889-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://889-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**